.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: go
 go:
-- 1.7
```

Comment:
I would like to keep 1.7 too for the time being, the issue was fixed in the swag repo